configs/razor_subnets/rtmdet_tiny_ofa_lat31_syncbn_16xb16-300e_coco.py

```diff
@@ -0,0 +1,142 @@
+_base_ = [
+    'mmrazor::_base_/nas_backbones/ofa_mobilenetv3_supernet.py',
+    '../rtmdet/rtmdet_s_syncbn_fast_8xb32-300e_coco.py'
+]
+
+custom_imports = dict(imports=['mmrazor.models'], allow_failed_imports=False)
```

Comment:
Why add this line?
MMEngine will register all MMRazor's module

Reply:
Oops, fixed :)